### PR TITLE
Implement support for fastify lightmyrequest responses

### LINKED ISSUE #44: Fastify support
Currently `api-contract-validator` does not support response format from fastify (based on light-my-request).

### PR DESCRIPTION
**Pull request by kibertoad**
fixes #44

### CODE REVIEW

File: lib/helpers/coverage.js [hunk 1]
```diff
@@ -35,6 +35,11 @@ module.exports.init = ({
 };
 
 module.exports.setCoverage = ({ path, method, status }) => {
+  if (!schema) {
```
Comment (kibertoad) [inline lib/helpers/coverage.js:38]: This used to break individual tests unless they initted coverage explicitly as well

**Comment by kibertoad** [comment]
@kobik Coverage drop doesn't seem to be significant, it's very much for handling not very relevant edge cases.

File: lib/helpers/coverage.js [hunk 2]
```diff
@@ -35,6 +35,11 @@ module.exports.init = ({
...
+    console.warn('Coverage was not initted. If you want to calculate coverage, make sure to call init() on coverage helper. Skipping coverage calculation');
```
Comment (kobik) [inline lib/helpers/coverage.js:39]: did you mean `initiated` instead of `initted`?

Comment (kibertoad) [reply lib/helpers/coverage.js]: fixed

**Comment by kibertoad** [comment]
@kobik Anything else?

**Comment by kobik** [comment]
@kibertoad, is lightmyrequest something like nock?

**Comment by kibertoad** [comment]
@kobik It's a library that fastify uses internally to simulate requests. It's closer to supertest, as its purpose is to enable `fastifyApp.inject().get('/endpointName')` kind of requests.
